docs/pages/identity-governance/entra-id.mdx

```diff
@@ -0,0 +1,359 @@
+---
+title: Entra ID Integration
+description: Describes how to set up the Teleport Entra ID integration in Teleport.
+labels:
+ - how-to
+ - identity-governance
+---
+
+Entra ID integration enables the following features in Teleport:
+
+1. **Single sign-on (SSO):** Configures Teleport authentication with Entra ID as an
+   identity provider.
+2. **User sync:** Imports Entra ID users as Teleport users.
+3. **Group sync:** Imports Entra ID groups as Teleport
+   Access Lists. 
+4. **Application sync (Optional):** Imports Entra ID applications. 
+5. **Integration with Teleport Identity Security (Optional):** Lets you analyze user access paths
+and policies from the Teleport Identity Security product. 
+
+## How it works
+
+An enterprise application will be configured in the Entra ID. 
+The purpose of this application is twofold - 
+1. Set up a SAML SSO provider for Teleport. 
+2. Set up Microsoft graph API credential for Teleport that allows to fetch resources 
+from Entra ID directory to Teleport.  
+
+Once the integration is configured, Teleport periodically imports users, groups, 
+and applications into Teleport.
+
+The installation process provides an option to extend Entra ID integration with the 
+Teleport Identity Security product, which offers user [access path and policy analysis](../identity-security/integrations/entra-id.mdx).
+
+### Choosing an authentication method
+
+In order to fetch resources from the Entra ID directory, Teleport needs to be 
+configured with credential that grants access to the Microsoft Graph API. 
+
+#### Teleport as an OIDC Provider for Entra ID 
+
+In this setup, Teleport is configured as an OpenID Connect (OIDC) identity provider, 
+establishing a trusted connection with an Entra ID application created during setup. 
+This trust allows Teleport to authenticate using the Entra ID application, accessing 
+permissions tied to it without requiring additional credentials or managed Identities.
+
+Direct bidirectional connectivity between Teleport and Azure is necessary for Azure to 
+validate the OIDC tokens issued by Teleport.
+
+#### System credentials
+
+Designed for air-gapped Teleport clusters that are not publicly accessible, this setup 
+accommodates environments where Azure cannot validate OIDC tokens issued by Teleport. 
+Instead, Teleport relies on Azure credentials available on the VMs where Teleport 
```

Comment:
I think we should rephrase this to avoid referring VMs, first because it works with kubernetes pods and second because we can make those credentials available in AWS/GCP machines